source/trunk/Esf.sln

```diff
@@ -0,0 +1,65 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 14
```

Comment:
It's good to have separate repositories for big resources - like dlls, third party dependencies that we need to version as they will evolve slow compared to our code base.
Not sure if this is good idea for the plannig documents. The bin folder of the concept project shouldn't be in source control as this is output. We can keep it in separate snapshot(drop) repository.